src/sentry/processing/realtime_metrics/base.py

```diff
@@ -106,6 +110,7 @@ def add_project_to_lpq(self, project_id: int) -> bool:
         """
         raise NotImplementedError
 
+    # TODO(flub): See if this can be removed
```

Comment:
arguably i shouldn't have left this comment here.  but it seems you should just be able to use the `remove_projects_from_lpq()` call instead, I don't think there's anything that this signature gives you that you can't do from the other one.